styles/components/_buttons.scss

```diff
@@ -0,0 +1,9 @@
+// stub, should be fleshed out
+.cmp-btn {
```

Comment:
I think I would rename this to `.cmp-button` and change the filename to `_button.scss`. I think that would match our naming convention for components a little better.